test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -37,6 +37,80 @@
    :ssl-key     localhost-key
    :ssl-ca-cert ca-cert})
 
+(def script-to-check-if-constant-is-defined
+  "begin; InstanceID; true; rescue NameError; false; end")
```

Comment:
To avoid the Wolfe Principle violation, could we do something like this?

~~~~
"if (defined? InstanceID); InstanceID; else; false; end"
~~~~